docs/modules/appendix/pages/glossary.adoc

```diff
@@ -201,6 +206,10 @@ Recurring fine::
   Recurring Fine is the official term for daily or other regularly accruing overdue fines. 
 Register Patron::
   The process of adding a Patron account with in Evergreen.
+Reindex::
+  To recreate or update the index.  See <<Index>>.
```

Comment:
This cross-reference doesn't quite work, since it is looking for the `Index` anchor (capitalized), but the actual anchor is called `index` (not capitalized).